### PR TITLE
feat(get-started): v2

### PR DESCRIPTION
**Pull request by ariellalgilmore**
### Related Ticket(s)

#1674 
### Description

just removed the tutorial section because those video's might be confusing since they are v1. I didn't reference Carbon's developer get started because i also think that would be confusing about carbon vs carbon for ibm.com and the get started process is a bit different
### Changelog

**Changed**

- also updated new office hour info for developers

### CODE REVIEW

**Comment by oliviaflory** [comment]
@ariellalgilmore thank you, this looks good

Would you mind going through the Getting started page and cleaning up how we write out Web Components? Right now we have a mix of:
- Web Components
- Web components
- web components

I'm assuming capitalizing both `Web Components` is the ideal because that's how it's written on the Frameworks tab on our site and on Carbon's site, along with the links to Storybook on both sites.

Screenshot 2023-11-01 at 5 44 44 PM

File: src/pages/developing/index.mdx
Comment (oliviaflory) [inline src/pages/developing/index.mdx:1]: > Carbon for IBM.com is available in <Link to="/developing/frameworks/web-components">web components</Link>, or <Link to="/developing/frameworks/react-wrapper">React wrapper</Link> for web components. Choosing a framework would depend on the needs of your application.

Should we re-phrase this to be:
_Carbon for IBM.com is available in <Link to="/developing/frameworks/web-components">Web Components</Link>, and <Link to="/developing/frameworks/react-wrapper">Web Components: React wrapper</Link>. Choosing a framework would depend on the needs of your application._

Comment (ariellalgilmore) [reply src/pages/developing/index.mdx:1]: updated!

**Review by oliviaflory — APPROVED** [review]
Thanks @ariellalgilmore

**Pull request merged**